packages/universe-components/src/donationForm/DonationForm.tsx

```diff
@@ -36,14 +38,17 @@ interface DonationFormProps {
   }>;
 }
 
+const starknetAddress: string = appConfig.chains.starknet?.network === "mainnet" 
```

Comment:
we should actually only have these on staging and development. In fact, this could be part of the appConfig, an optional chainDefaults.defaultDonationAddress or something, but only on staging and development (maybe we can throw an error if its on the production config).

Reply:
The risk is: if a real organization signs up and doesn't set up a wallet, we could receive funds on their behalf, and we don't want that.

Reply:
I absolutely agree we are not supposed to have this in production. I have implemented a fix as suggested.